src/modules/notifications/NotificationsManager.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+import messaging from '@react-native-firebase/messaging'
+
+const requestUserPermissionForPushNotifications = async () => {
+  const authStatus = await messaging().requestPermission()
+
+  const enabled =
+    authStatus === messaging.AuthorizationStatus.AUTHORIZED ||
+    authStatus === messaging.AuthorizationStatus.PROVISIONAL
+
+  if (enabled) {
+    console.debug('Authorization status:', authStatus)
+  }
+}
+
+const unSubscribeToPreviousCityTopic = async (prevCity, prevLanguage) => {
```

Comment:
```suggestion
const unsubscribeFromCity = async (prevCity, prevLanguage) => {
```

Reply:
I think i will keep it `unsubscribeFromPreviousCity`